<!DOCTYPE html><html lang="en" >


<head>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  
  
  
    <meta name="generator" content="Wowchemy 5.4.0 for Hugo" />
  

  
  









  




  
  

  
  
  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Gillian P. McHugo" />

  
  
  
    
  
  <meta name="description" content="Conservation of wild and livestock/aquaculture species aims at maximizing genetic diversity on the long-term, while selection programs focus mainly on genetic improvement in the next generation. Both genetic drift and selection can lead to high rates at which inbreeding increases if a small number of individuals contribute largely to subsequent generations. Increases in the rate of inbreeding would have negative consequences such as reductions of genetic variability and increased risk of inbreeding depression that may affect specific genomic regions and that can compromise the success of both conservation and selection programs. Traditionally, the control of inbreeding has been tracked by managing the relationships between individuals using pedigree records. With the advent of new genotyping and high-throughput sequencing technologies, obtaining more precise information about the relationships between individuals is now possible. While single nucleotide polymorphisms (SNP) chips or genotyping-by-sequencing data contain only a subset of the polymorphic variants available in a species, whole genome sequencing data provide access to complete information on all the variants of an individual. As a consequence, the estimation of coefficients to measure genetic variability may be affected by the type of data used for their estimation.&lt;br&gt;&lt;br&gt;During the post-genomics era, a variety of methods to characterize the levels of genetic diversity from genomic information within and between populations have been developed. These are related to the estimation of the selection signature and population structure or coefficients to determine the increase in inbreeding such as effective population size. However, under this new genomic perspective, methods and concepts need to be revised. In parallel to the development of genomic tools, traditional and new strategies to control inbreeding to simultaneously target conservation of genetic diversity and genetic improvement need to be developed or accommodated to the new context. The availability of genomic information opens up possibilities to include genome wide marker information in prediction of total breeding values and to obtain higher genetic gains through genomic selection but controlling the increase of the rate of inbreeding is also necessary to ensure the viability of the programs." />

  
  <link rel="alternate" hreflang="en" href="https://gillianmchugo.github.io/publication/saura-et-al-2020-frontiers-in-genetics/" />

  
  
  
    <meta name="theme-color" content="#008695" />
  

  
  

  

  <link rel="stylesheet" href="/css/vendor-bundle.min.f1ecf783c14edc00c9320c205831ad8e.css" media="print" onload="this.media='all'">

  
  
  
    
    
      <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha512-W0xM4mr6dEP9nREo7Z9z+9X70wytKvMGeDsj7ps2+xg5QPrEBXC8tAW1IFnzjR6eoJ90JmCnFzerQJTLzIEHjA==" crossorigin="anonymous" media="print" onload="this.media='all'">
    

    
    
    
    
      
      
    
    
    

    
    
    
      
    
    

    
    
    

    

    
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
      
      

      
      
        
      

      
    
      
      

      
      

      
    
      
      

      
      

      
    
  

  
  
  
  
  
  <link rel="stylesheet" href="/css/wowchemy.4cc17defc2536beff049cd123315335e.css" />

  



  

  

  




  
  
  

  

  
    <link rel="manifest" href="/manifest.webmanifest" />
  

  <link rel="icon" type="image/png" href="/media/icon_hu743880228473ec0451707c5fe015dc10_9113_32x32_fill_lanczos_center_3.png" />
  <link rel="apple-touch-icon" type="image/png" href="/media/icon_hu743880228473ec0451707c5fe015dc10_9113_180x180_fill_lanczos_center_3.png" />

  <link rel="canonical" href="https://gillianmchugo.github.io/publication/saura-et-al-2020-frontiers-in-genetics/" />

  
  
  
  
  
  
  
  
    
    
  
  

  
  
    
    
  
  <meta property="twitter:card" content="summary" />
  
  <meta property="og:site_name" content="Gillian McHugo" />
  <meta property="og:url" content="https://gillianmchugo.github.io/publication/saura-et-al-2020-frontiers-in-genetics/" />
  <meta property="og:title" content="Genetic variability in conservation and selection programs in the post-genomics era | Gillian McHugo" />
  <meta property="og:description" content="Conservation of wild and livestock/aquaculture species aims at maximizing genetic diversity on the long-term, while selection programs focus mainly on genetic improvement in the next generation. Both genetic drift and selection can lead to high rates at which inbreeding increases if a small number of individuals contribute largely to subsequent generations. Increases in the rate of inbreeding would have negative consequences such as reductions of genetic variability and increased risk of inbreeding depression that may affect specific genomic regions and that can compromise the success of both conservation and selection programs. Traditionally, the control of inbreeding has been tracked by managing the relationships between individuals using pedigree records. With the advent of new genotyping and high-throughput sequencing technologies, obtaining more precise information about the relationships between individuals is now possible. While single nucleotide polymorphisms (SNP) chips or genotyping-by-sequencing data contain only a subset of the polymorphic variants available in a species, whole genome sequencing data provide access to complete information on all the variants of an individual. As a consequence, the estimation of coefficients to measure genetic variability may be affected by the type of data used for their estimation.&lt;br&gt;&lt;br&gt;During the post-genomics era, a variety of methods to characterize the levels of genetic diversity from genomic information within and between populations have been developed. These are related to the estimation of the selection signature and population structure or coefficients to determine the increase in inbreeding such as effective population size. However, under this new genomic perspective, methods and concepts need to be revised. In parallel to the development of genomic tools, traditional and new strategies to control inbreeding to simultaneously target conservation of genetic diversity and genetic improvement need to be developed or accommodated to the new context. The availability of genomic information opens up possibilities to include genome wide marker information in prediction of total breeding values and to obtain higher genetic gains through genomic selection but controlling the increase of the rate of inbreeding is also necessary to ensure the viability of the programs." /><meta property="og:image" content="https://gillianmchugo.github.io/media/icon_hu743880228473ec0451707c5fe015dc10_9113_512x512_fill_lanczos_center_3.png" />
    <meta property="twitter:image" content="https://gillianmchugo.github.io/media/icon_hu743880228473ec0451707c5fe015dc10_9113_512x512_fill_lanczos_center_3.png" /><meta property="og:locale" content="en" />
  
    
      <meta
        property="article:published_time"
        content="2020-12-22T00:00:00&#43;00:00"
      />
    
    <meta property="article:modified_time" content="2020-12-22T00:00:00&#43;00:00">
  

  


    









<script type="application/ld+json">
{
  "@context": "https://schema.org",
  "@type": "Article",
  "mainEntityOfPage": {
    "@type": "WebPage",
    "@id": "https://gillianmchugo.github.io/publication/saura-et-al-2020-frontiers-in-genetics/"
  },
  "headline": "Genetic variability in conservation and selection programs in the post-genomics era",
  
  "datePublished": "2020-12-22T00:00:00Z",
  "dateModified": "2020-12-22T00:00:00Z",
  
  "author": {
    "@type": "Person",
    "name": "Maria Saura"
  },
  
  "publisher": {
    "@type": "Organization",
    "name": "Gillian McHugo",
    "logo": {
      "@type": "ImageObject",
      "url": "https://gillianmchugo.github.io/media/icon_hu743880228473ec0451707c5fe015dc10_9113_192x192_fill_lanczos_center_3.png"
    }
  },
  "description": "Conservation of wild and livestock/aquaculture species aims at maximizing genetic diversity on the long-term, while selection programs focus mainly on genetic improvement in the next generation. Both genetic drift and selection can lead to high rates at which inbreeding increases if a small number of individuals contribute largely to subsequent generations. Increases in the rate of inbreeding would have negative consequences such as reductions of genetic variability and increased risk of inbreeding depression that may affect specific genomic regions and that can compromise the success of both conservation and selection programs. Traditionally, the control of inbreeding has been tracked by managing the relationships between individuals using pedigree records. With the advent of new genotyping and high-throughput sequencing technologies, obtaining more precise information about the relationships between individuals is now possible. While single nucleotide polymorphisms (SNP) chips or genotyping-by-sequencing data contain only a subset of the polymorphic variants available in a species, whole genome sequencing data provide access to complete information on all the variants of an individual. As a consequence, the estimation of coefficients to measure genetic variability may be affected by the type of data used for their estimation.\u003cbr\u003e\u003cbr\u003eDuring the post-genomics era, a variety of methods to characterize the levels of genetic diversity from genomic information within and between populations have been developed. These are related to the estimation of the selection signature and population structure or coefficients to determine the increase in inbreeding such as effective population size. However, under this new genomic perspective, methods and concepts need to be revised. In parallel to the development of genomic tools, traditional and new strategies to control inbreeding to simultaneously target conservation of genetic diversity and genetic improvement need to be developed or accommodated to the new context. The availability of genomic information opens up possibilities to include genome wide marker information in prediction of total breeding values and to obtain higher genetic gains through genomic selection but controlling the increase of the rate of inbreeding is also necessary to ensure the viability of the programs."
}
</script>

  

  

  

  





  <title>Genetic variability in conservation and selection programs in the post-genomics era | Gillian McHugo</title>
</head>


<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" class="page-wrapper   " data-wc-page-id="4a874f3bfa2ffcf11c904e17039a17b8" >

  
  
  
  
  
  
  
  
  
  <script src="/js/wowchemy-init.min.8f76bdc9e086322ed5147724ebba3d06.js"></script>

  


<aside class="search-modal" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#" aria-label="Close"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search" class="form-control"
        aria-label="Search...">
        
      </div>

      
      

      

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>



  <div class="page-header">
    












<header class="header--fixed">
  <nav class="navbar navbar-expand-lg navbar-light compensate-for-scrollbar" id="navbar-main">
    <div class="container-xl">

      
      <div class="d-none d-lg-inline-flex">
        <a class="navbar-brand" href="/">Gillian McHugo</a>
      </div>
      

      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar-content" aria-controls="navbar-content" aria-expanded="false" aria-label="Toggle navigation">
      <span><i class="fas fa-bars"></i></span>
      </button>
      

      
      <div class="navbar-brand-mobile-wrapper d-inline-flex d-lg-none">
        <a class="navbar-brand" href="/">Gillian McHugo</a>
      </div>
      

      
      
      <div class="navbar-collapse main-menu-item collapse justify-content-end" id="navbar-content">

        
        <ul class="navbar-nav d-md-inline-flex">
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#about"><span>About</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#publications"><span>Publications</span></a>
          </li>

          
          

          

          
          
          
            
          

          

          
          
          
          

          
            
              
              
            
            
              
              
              
                
              
              
            
          

          <li class="nav-item">
            <a class="nav-link " href="/#contact"><span>Contact</span></a>
          </li>

          
          

          
          <li class="nav-item dropdown">
            <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"><span>CV</span><span class="caret"></span>
            </a>
            <div class="dropdown-menu">
              
                <a class="dropdown-item" href="https://raw.githubusercontent.com/gillianmchugo/cv/main/GillianMcHugoCV.pdf"><span>Download</span></a>
              
                <a class="dropdown-item" href="/cv"><span>View</span></a>
              
            </div>
          </li>

          
          

          
          <li class="nav-item dropdown">
            <a href="#" class="nav-link dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"><span>More</span><span class="caret"></span>
            </a>
            <div class="dropdown-menu">
              
                <a class="dropdown-item" href="/publication"><span>All publications</span></a>
              
                <a class="dropdown-item" href="/acknowledgement"><span>Acknowledgements</span></a>
              
            </div>
          </li>

          
          

        

          
        </ul>
      </div>

      <ul class="nav-icons navbar-nav flex-row ml-auto d-flex pl-md-2">

        
        
          
        

        
        
        <li class="nav-item">
          <a class="nav-link js-search" href="#" aria-label="Search"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        
        
        <li class="nav-item dropdown theme-dropdown">
          <a href="#" class="nav-link" data-toggle="dropdown" aria-haspopup="true" aria-label="Display preferences">
            <i class="fas fa-moon" aria-hidden="true"></i>
          </a>
          <div class="dropdown-menu">
            <a href="#" class="dropdown-item js-set-theme-light">
              <span>Light</span>
            </a>
            <a href="#" class="dropdown-item js-set-theme-dark">
              <span>Dark</span>
            </a>
            <a href="#" class="dropdown-item js-set-theme-auto">
              <span>Automatic</span>
            </a>
          </div>
        </li>
        

        
        

      </ul>

    </div>
  </nav>
</header>


  </div>

  <div class="page-body">
    








<div class="pub">

  












  

  
  
  
<div class="article-container pt-3">
  <h1>Genetic variability in conservation and selection programs in the post-genomics era</h1>

  

  
    


<div class="article-metadata">

  
  
  
  
  <div>
    

  <span >
      Maria Saura</span>, <span >
      Francesca Bertolini</span>, <span >
      Christos Palaiokostas (Eds)</span>
  </div>
  
  

  
  <span class="article-date">
    
    
      
    
    2020
  </span>
  

  

  

  
  
  
  
  
  

  
  

</div>

    




<div class="btn-links mb-3">
  
  








  





<a href="#" class="btn btn-outline-primary btn-page-header js-cite-modal"
        data-filename="/publication/saura-et-al-2020-frontiers-in-genetics/cite.bib">
  Cite
</a>













<a class="btn btn-outline-primary btn-page-header" href="https://doi.org/10.3389/978-2-88966-249-4" target="_blank" rel="noopener">
  DOI
</a>


  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://www.frontiersin.org/research-topics/8876/genetic-variability-in-conservation-and-selection-programs-in-the-post-genomics-era" target="_blank" rel="noopener">
    <i class="fas fa-external-link-square-alt mr-1"></i>Link</a>

  
  
  
    
  
  
  
  
  
    
  
  <a class="btn btn-outline-primary btn-page-header" href="https://www.frontiersin.org/research-topics/8876/pdf" target="_blank" rel="noopener">
    <i class="fas fa-file-pdf mr-1"></i>Download</a>


</div>


  
</div>



  <div class="article-container">

    
    <h3>Abstract</h3>
    <p class="pub-abstract">Conservation of wild and livestock/aquaculture species aims at maximizing genetic diversity on the long-term, while selection programs focus mainly on genetic improvement in the next generation. Both genetic drift and selection can lead to high rates at which inbreeding increases if a small number of individuals contribute largely to subsequent generations. Increases in the rate of inbreeding would have negative consequences such as reductions of genetic variability and increased risk of inbreeding depression that may affect specific genomic regions and that can compromise the success of both conservation and selection programs. Traditionally, the control of inbreeding has been tracked by managing the relationships between individuals using pedigree records. With the advent of new genotyping and high-throughput sequencing technologies, obtaining more precise information about the relationships between individuals is now possible. While single nucleotide polymorphisms (SNP) chips or genotyping-by-sequencing data contain only a subset of the polymorphic variants available in a species, whole genome sequencing data provide access to complete information on all the variants of an individual. As a consequence, the estimation of coefficients to measure genetic variability may be affected by the type of data used for their estimation.<br><br>During the post-genomics era, a variety of methods to characterize the levels of genetic diversity from genomic information within and between populations have been developed. These are related to the estimation of the selection signature and population structure or coefficients to determine the increase in inbreeding such as effective population size. However, under this new genomic perspective, methods and concepts need to be revised. In parallel to the development of genomic tools, traditional and new strategies to control inbreeding to simultaneously target conservation of genetic diversity and genetic improvement need to be developed or accommodated to the new context. The availability of genomic information opens up possibilities to include genome wide marker information in prediction of total breeding values and to obtain higher genetic gains through genomic selection but controlling the increase of the rate of inbreeding is also necessary to ensure the viability of the programs.</p>
    

    
    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Type</div>
          <div class="col-12 col-md-9">
            <a href="/publication/#7">
              Other
            </a>
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Publication</div>
          <div class="col-12 col-md-9">Research Topic. Frontiers</div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    <div class="space-below"></div>

    <div class="article-style">




<html>
    <style>
        section {
            background: white;
            color: black;
            border-radius: 1em;
            left: 50% }
        #inner {
            display: inline-block;
            display: flex;
            justify-content: center }
    </style>
    <section>
        <div id="inner">
        <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>
            <span style="float:left"; 
            class="__dimensions_badge_embed__" 
            data-doi="10.3389/978-2-88966-249-4" 
            data-hide-zero-citations="true" 
            data-legend="always" 
            data-style="small_circle">
            </span>
        <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
            <div  style="float:right"; 
            data-badge-details="right" 
            data-badge-type="donut"
            data-doi="10.3389/978-2-88966-249-4"   
            data-condensed="true" 
            data-hide-no-mentions="true" 
            class="altmetric-embed">
            </div>
        </div>
    </section>

</div>

    








<div class="share-box">
  <ul class="share">
    
      
      
      
        
      
      
      
      <li>
        <a href="https://twitter.com/intent/tweet?url=https://gillianmchugo.github.io/publication/saura-et-al-2020-frontiers-in-genetics/&amp;text=Genetic%20variability%20in%20conservation%20and%20selection%20programs%20in%20the%20post-genomics%20era" target="_blank" rel="noopener" class="share-btn-twitter" aria-label="twitter">
          <i class="fab fa-twitter"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="mailto:?subject=Genetic%20variability%20in%20conservation%20and%20selection%20programs%20in%20the%20post-genomics%20era&amp;body=https://gillianmchugo.github.io/publication/saura-et-al-2020-frontiers-in-genetics/" target="_blank" rel="noopener" class="share-btn-email" aria-label="envelope">
          <i class="fas fa-envelope"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.linkedin.com/shareArticle?url=https://gillianmchugo.github.io/publication/saura-et-al-2020-frontiers-in-genetics/&amp;title=Genetic%20variability%20in%20conservation%20and%20selection%20programs%20in%20the%20post-genomics%20era" target="_blank" rel="noopener" class="share-btn-linkedin" aria-label="linkedin-in">
          <i class="fab fa-linkedin-in"></i>
        </a>
      </li>
    
  </ul>
</div>











  
  
    




  
    




  
    




  


















  </div>
</div>
  </div>

  <div class="page-footer">
    
    
    <div class="container">
      <footer class="site-footer">

  



  

  

  

  
  






  <p class="powered-by">
    
    
    
      
      
      
      
      
      
      Published with <a href="https://wowchemy.com/?utm_campaign=poweredby" target="_blank" rel="noopener">Wowchemy</a> — the free, <a href="https://github.com/wowchemy/wowchemy-hugo-themes" target="_blank" rel="noopener">open source</a> website builder that empowers creators.
    
  </p>
</footer>

    </div>
    
  </div>

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

      

    
    <script src="/js/vendor-bundle.min.b73dfaac3b6499dc997741748a7c3fe2.js"></script>

    
    
    
      
      
        <script src="https://cdn.jsdelivr.net/gh/desandro/imagesloaded@v4.1.4/imagesloaded.pkgd.min.js" integrity="sha512-S5PZ9GxJZO16tT9r3WJp/Safn31eu8uWrzglMahDT4dsmgqWonRY9grk3j+3tfuPr9WJNsfooOR7Gi7HL5W2jw==" crossorigin="anonymous"></script>
        <script src="https://cdn.jsdelivr.net/gh/metafizzy/isotope@v3.0.6/dist/isotope.pkgd.min.js" integrity="sha512-Zq2BOxyhvnRFXu0+WE6ojpZLOU2jdnqbrM1hmVdGzyeCa1DgM3X5Q4A/Is9xA1IkbUeDd7755dNNI/PzSf2Pew==" crossorigin="anonymous"></script>
      

      
      

      

      

    

    
    
    

    
    

    
    
    
      
      <script id="search-hit-fuse-template" type="text/x-template">
        <div class="search-hit" id="summary-{{key}}">
          <div class="search-hit-content">
            <div class="search-hit-name">
              <a href="{{relpermalink}}">{{title}}</a>
              <div class="article-metadata search-hit-type">{{type}}</div>
              <p class="search-hit-description">{{snippet}}</p>
            </div>
          </div>
        </div>
      </script>
      
        <script src="https://cdn.jsdelivr.net/gh/krisk/Fuse@v3.2.1/dist/fuse.min.js" integrity="sha512-o38bmzBGX+hD3JHWUFCDA09btWaqrNmoJ3RXLlrysA7PP01Kgs4UlE4MhelE1v5dJR3+cxlR4qQlotsW7jKsnw==" crossorigin="anonymous"></script>
        <script src="https://cdn.jsdelivr.net/gh/julmot/mark.js@8.11.1/dist/jquery.mark.min.js" integrity="sha512-mhbv5DqBMgrWL+32MmsDOt/OAvqr/cHimk6B8y/bx/xS88MVkYGPiVv2ixKVrkywF2qHplNRUvFsAHUdxZ3Krg==" crossorigin="anonymous"></script>
      
    

    
    

    
    
    
    

    
    
      
      
      
      
      
      
      
    

    

    
    
    
    <script id="page-data" type="application/json">{"use_headroom":true}</script>

    
    
      <script src="/js/wowchemy-headroom.1cb9e2fc8399acee94eab837265b73bf.js" type="module"></script>
    
    
    
    
    
    
    
      
      
    
    
    <script src="/en/js/wowchemy.min.098b77d0303ddc87a1a035cfc5920628.js"></script>

    






</body>
</html>
